python/lsst/meas/deblender/deblend.py

```diff
@@ -908,14 +904,15 @@ def deblend(self, exposures, sources, psfs, bands=None):
         from lsst.meas.deblender.baseline import newDeblend
         import deblender
 
-        if bands is None:
-            bands = list(exposures.keys())
-        maskedImages = {band:exp.getMaskedImage() for band, exp in exposures.items()}
-        self.log.info("Deblending {0} sources in {1} exposures".format(len(sources), len(bands)))
+        filters = mExposure.filters
+        mMaskedImage = afwImage.MultibandMaskedImage(filters=mExposure.filters, image=mExposure.image,
+                                                     mask=mExposure.mask, variance=mExposure.variance)
```

Comment:
Hmm, so there isn't a constructor for `MultibandMaskedImage` that takes a `MultibandExposure`? That seems sad. Although perhaps you don't want that because it is a narrowing conversion? Then perhaps a static method? E.g. `mMaskedImage = afwImage.MultibandMaskedImage.fromMultibandExposure(mExposure)`? Or a property (`mMaskedImage = mExposure.multibandMaskedImage`)? both would reduce the potential for swapping `image/mask/variance` by accident.

Reply:
I see your point. Actually, I *think* that `MultibandMaskedImage.fromImages` would actually work if a `MultibandExposure` is used in place of the of a `MultibandMaskedImage`. I'll open a ticket to rename `fromImages` to a more inclusive name.